src/postgres/tree/IPostgreSQLTreeRoot.ts

```diff
@@ -0,0 +1,11 @@
+
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ISubscriptionContext } from "vscode-azureextensionui";
+
+export interface IPostgreSQLTreeRoot extends ISubscriptionContext {
+    isEmulator: boolean;
```

Comment:
Uh does postgres support an emulator? I think that's Cosmos specific, in which case you can remove all emulator code from Postgres

Reply:
IPostgreSQLTreeRoot only contains isEmulator for now, so it will be empty once I remove that variable. Should I keep this class for any future additions or just use the super-type for now in its place?

Reply:
Just use the super-type